main_routers/jukebox_router.py

```
@@ -0,0 +1,1156 @@
+        user_data = {
+            "version": self.data.get("version", "1.0"),
+            "songs": user_songs,
+            "actions": user_actions,
+            "bindings": {},  # 保存所有涉及用户资源的绑定
+            "md5Index": {
+                "songs": {},
+                "actions": {}
+            }
+        }
+        
+        # 保存绑定关系：只要绑定涉及用户资源就保存
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)